Gems/ScriptCanvas/Code/Include/ScriptCanvas/Internal/Nodes/ExpressionNodeBase.h

```diff
@@ -67,6 +68,8 @@ namespace ScriptCanvas
                 void OnSlotRemoved(const SlotId& slotId) override;
 
                 bool OnValidateNode(ValidationResults& validationResults) override;
+
+                void FixDynamicGroupDataType(const SlotId& slotId,Data::Type displayType);
```

Comment:
Is there a better name than "Fix" ? it's not clear reading the name what is wrong with the DynamicGroupDataType that needs to be fixed, it's a bit ambiguous, is it fix as in "to correct" or fix as in "to lock into place" ?